System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -10,20 +10,18 @@
 #else
 using nuint = System.UInt32;
 #endif 
-namespace System.IO.Compression.Brotli
+namespace System.IO.Compression
 {
     public partial class BrotliStream : Stream
```

Comment:
Yes, that makes sense, but we decided to start with primitive realization using buffers and after that start to work with Spans.